test/core/memory_grow.wast

```diff
@@ -0,0 +1,25 @@
+;; Test that newly allocated memory (program start and memory.grow) is zeroed
```

Comment:
Currently this only tests memory.grow. To also test program start, you could declare the memory with a size > 0.